content/tutorials/git_authentication/index.Rmd

```diff
@@ -0,0 +1,266 @@
+
+Personal access tokens can only be used for HTTPS Git operations.
+When you create a PAT, you will be asked which scopes should be allowed (i.e. what rights do you give).
+We suggest that you stick with the scopes recommended by the `usethis` package.
+Executing the following command takes your browser to a pre filled form with the recommended scopes (repo, user, workflow) where you can create your PAT.
+
+```{r eval=FALSE}
+usethis::create_github_token()
+```
+
+After you have done this, you can store this PAT in the Git Credential Manager using this code:
+
+```{r eval=FALSE}
+gitcreds::gitcreds_set()
+```
+
+which will open a prompt where you can paste your PAT (or replace an old one).
+
+If you want to know more about PATs read [this section from happy git with R](https://happygitwithr.com/https-pat.html#get-a-pat) and [this section from the usethis package](https://usethis.r-lib.org/articles/git-credentials.html#get-a-personal-access-token-pat).
+
+After you've done this, you can check your authentication settings.
+The `usethis` package has a function to get a situation report on your current Git/GitHub status, including information about authentication:
+
+```{r eval=FALSE}
+usethis::git_sitrep()
+```
+
+If you see warnings about git_vaccinated, it is recommended to execute
+
+```{r}
+usethis::git_vaccinate()
+```
+
+which adds a global `.gitignore` file that decreases the chance that you accidentally leak credentials.
+
+The situation report should normally report that the default Git protocol is `https`, which we will discuss next.
+
+## <a name="https"></a>Using the HTTPS protocol
+
+The HTTPS protocol is the GitHub default URL transport protocol that uses the Transport Layer Security (TLP) encryption protocol to encrypt communications.
+To learn more about what https is, [read this explanation](https://www.cloudflare.com/learning/ssl/what-is-https/).
+
+The usual way to choose the protocol is when you clone a repository to your computer.
+You can do this in various ways:
+
+- Manual way:
+    - Go to https://github.com/inbo/tutorials press clone button and copy the URL (default is HTTPS: https://github.com/inbo/tutorials.git)
+    - Open RStudio -> File -> New project -> Version control -> Git -> paste https://github.com/inbo/tutorials.git -> create project
+- Using git commands
+    - Open a terminal
+    - (Go to the folder where you want to clone the repository using `cd` command or open the terminal directly at that location)
+    - type `git clone https://github.com/inbo/tutorials.git`
+- Using R packages
+    - `usethis::create_from_github(repo_spec = "https://github.com/inbo/tutorials.git", destdir = "path/to/gitrepofolder")`
+    - `gert::git_clone(url = "https://github.com/inbo/tutorials.git", path = "path/to/gitrepofolder")`
+
+See also in [SSH protocol](#ssh) the paragraph about the use of the `git remote` command. 
```

Comment:
The `#ssh` fragment is non-functional.

Reply:
See [918729b](https://github.com/inbo/tutorials/pull/309/commits/918729b1fba56ce55c446cd00bea10b97d325aed) (need to check if it works)